robottelo/bz_helpers.py

```diff
@@ -15,7 +16,22 @@ def get_decorated_bugs():  # pragma: no cover
     Important information is stored on `bug_data` key::
         bugs[BUG_ID]['bug_data']['resolution|status|flags|whiteboard']
     """
-    parser = Parser(BASE_PATH, filters=[BZDecorator])
+    # look for settings bugzilla credentials
+    # any way Parser bugzilla reader will check for exported environment names
+    # BUGZILLA_USER_NAME and BUGZILLA_USER_PASSWORD if no credentials was
+    # supplied
+    bz_reader_options = {}
+    # we are at early stage and settings are not configured yet
+    settings.configure()
+    bz_credentials = settings.bugzilla.get_credentials()
+    if any(value is None for value in bz_credentials.values()):
```

Comment:
Please use `setting_is_set()`, and, basically, you can drop `settings.configure` as it's called in `setting_is_set()`.